From fd8530a2d9f41d5c2a8359e6a0efd2f836d0c359 Mon Sep 17 00:00:00 2001
From: Jordan Liggitt <liggitt@google.com>
Date: Sat, 22 May 2021 20:04:38 -0400
Subject: [PATCH] --EKS-PATCH-- (CHERRY-PICK OF #102224 FROM 1.22, DROP FROM
 1.22, 1.21.2+, 1.20.8+) Fix expired unit test certs

This patch was cherry-picked from upstream 1.22 PR #102224 so it can be
dropped from the EKS 1.22 branch.

In addition, it got cherry-picked into upstream 1.21.1+ #102225 and
1.20.8+ #102226 so it can be dropped from those EKS branches too.

Conflicts:
  staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

Resolved by removing getSingleCounterValueFromRegistry which doesn't
exist in 1.20
---
 .../k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go b/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go
index eab1a2844e4..845a777e6b1 100644
--- a/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go
+++ b/staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go
@@ -21,7 +21,6 @@ import (
 	"crypto/x509"
 	"fmt"
 	"io/ioutil"
-	"k8s.io/apiserver/pkg/server/dynamiccertificates"
 	"net/http"
 	"net/http/httptest"
 	"net/http/httputil"
@@ -33,6 +32,8 @@ import (
 	"sync/atomic"
 	"testing"
 
+	"k8s.io/apiserver/pkg/server/dynamiccertificates"
+
 	"golang.org/x/net/websocket"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
